atlasdb-api/src/main/java/com/palantir/atlasdb/spi/AtlasDbFactory.java

```diff
@@ -36,6 +36,17 @@ default KeyValueService createRawKeyValueService(
         return createRawKeyValueService(config, leaderConfig, Optional.empty(), DEFAULT_INITIALIZE_ASYNC);
     }
 
+    /**
+     * Creates a KeyValueService instance of the requested type.
```

Comment:
...`of the type requested according to the @param{config}`. (my javadoc syntax is probably wrong)

Reply:
actually, there's no such syntax. https://stackoverflow.com/questions/1667212/how-to-add-reference-to-a-method-parameter-in-javadoc. Changed the line to: `Creates a KeyValueService instance of the type requested according to the config parameter.`